autonomy/cli/analyse.py

```diff
@@ -160,13 +162,103 @@ def docstrings(ctx: Context, update: bool) -> None:
                 click.echo("No update needed.")
 
 
-@analyse_group.command(name="logs")
-@click.argument("file", type=click.Path(file_okay=True, dir_okay=False, exists=True))
-def parse_logs(file: str) -> None:
-    """Parse logs of an agent service."""
+@analyse_group.command("logs")
+@click.option(
+    "--from-dir",
+    "logs_dir",
+    type=click.Path(
+        exists=True,
+    ),
+    required=True,
+    help="Path to logs directory",
+)
+@click.option(
+    "--reset-db",
+    is_flag=True,
+    help="Path to logs directory",
+)
+@click.option(
+    "-a",
+    "--agent",
+    "agents",
+    type=str,
+    multiple=True,
+    help="Agent IDs to include in analysis",
+    required=True,
+)
+@click.option(
+    "--start-time",
+    type=str,
+    help=f"Start time in `{TIME_FORMAT_TEMPLATE}` format",
+)
+@click.option(
+    "--end-time",
+    type=str,
+    help=f"End time in `{TIME_FORMAT_TEMPLATE}` format",
+)
+@click.option(
+    "--log-level",
+    type=click.Choice(choices=LOGGING_LEVELS, case_sensitive=True),
+    help="Logging level.",
+)
+@click.option(
+    "--period",
+    type=int,
+    help="Period ID",
+)
+@click.option(
+    "--round",
+    "round_name",
+    type=str,
+    help="Round name",
+)
+@click.option(
+    "--behaviour",
+    "behaviour_name",
+    type=str,
+    help="Behaviour name filter",
+)
+@click.option(
+    "--fsm",
+    "fsm_path",
+    is_flag=True,
+    type=str,
+    help="Print only the FSM execution path",
+)
```

Comment:
is it possible to go ahead and add a flag -f
so that this will stream to the console?